docs/modules/ROOT/pages/advanced-how-tos/managing-multiple-versions.adoc

```diff
@@ -161,6 +161,30 @@ event == "push" && target_branch == "v1.0.0"
 
 * The `appstudio.openshift.io/application` and `appstudio.openshift.io/component` labels must be adjusted to specify the right Application and Component respectively. Failing to do this will cause builds of the pipeline to be associated with the wrong application or component.
 
+=== Create ImageRepositories for each component
+
+This is done automatically when working with the UI, and must be done manually when creating components through CLI means.
+This must be done for each component to authenticate with the output-registry when pushing built images.
+
+[source,yaml]
+.imagerepository.yaml
+----
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ImageRepository
+metadata:
+  labels:
+    appstudio.redhat.com/application: konflux-multibranch-sample-app-v1-0-0
+    appstudio.redhat.com/component: konflux-multibranch-sample-v1-0-0
+    image-controller.appstudio.redhat.com/update-component-image: 'true'
+  name: imagerepository-for-konflux-multibranch-sample-v1-0-0-konflux-multibranch-sample-v1-0-0
```

Comment:
should reflect change in application name

even though you can create it like this, it would be better to have in example show that by default it is constructed with app/component name

Reply:
you are correct, changed